sdk/resourcemanager/Proto.Client/README.MD

```diff
@@ -0,0 +1,3 @@
+# Proto Client
```

Comment:
Wes, what's today's usage of $Root\samples folder? This is like sample client we may want to keep it outside of SDK folder.

Reply:
The root samples folder is for cross-library samples. If we end up keeping this longer term we should probably move it into a samples folder under Azure.ResourceManager.Core/samples.

Reply:
this will be deleted before moving into master